.github/workflows/maven-verify.yml

```diff
@@ -170,6 +189,7 @@ jobs:
         jdk: ${{ fromJSON( inputs.jdk-matrix ) }}
         distribution: ${{ fromJSON( inputs.jdk-distribution-matrix ) }}
         exclude: ${{ fromJSON( inputs.matrix-exclude ) }}
+      max-parallel: ${{ inputs.max-parallel }}
```

Comment:
Please fix indent

Reply:
But the indent is right, the problem is that GH shows you only 3 lines above, these three lines are nested ones. It looks like this:

```
    strategy:
      fail-fast: ${{ inputs.verify-fail-fast }}
      matrix:
        os: ${{ fromJSON( inputs.os-matrix ) }}
        jdk: ${{ fromJSON( inputs.jdk-matrix ) }}
        distribution: ${{ fromJSON( inputs.jdk-distribution-matrix ) }}
        exclude: ${{ fromJSON( inputs.matrix-exclude ) }}
      **max-parallel: ${{ inputs.max-parallel }}**

    steps:
      - name: Should run
        id: should-run
```

Reply:
😄